README.md

```diff
@@ -1,5 +1,230 @@
+        .password("vault.stronghold") // A password to encrypt the stored data.WARNING: Never hardcode passwords in production code.
+        .build(PathBuf::from("vault.stronghold"))?; // The path to store the account snapshot.
+
+    let client_options = ClientOptions::new().with_node("https://api.testnet.shimmer.network")?;// The node to connect to.
+
+    // Set up and store the wallet.
+    let wallet = Wallet::builder()
+        .with_secret_manager(SecretManager::Stronghold(secret_manager))
+        .with_client_options(client_options)
+        .with_coin_type(SHIMMER_COIN_TYPE)
+        .finish()
+        .await?;
+
+    // Generate a mnemonic and store it in the Stronghold vault.
+    // INFO: It is best practice to back up the mnemonic somewhere secure.
+    let mnemonic = wallet.generate_mnemonic()?;
+    wallet.store_mnemonic(mnemonic).await?;
+
+    // Create an account.
+    let account = wallet
+        .create_account()
+        .with_alias("Alice".to_string()) // A name to associate with the created account.
+        .finish()
+        .await?;
+
+
+    Ok(())
+}
+```
+
+#### Client
+
+To use the Client module, you simply need to create a `Client`.
+
+
+```rust
+use iota_sdk::client::{
+    Client,
+};
+
+let client = Client::builder()
+    .with_node('https://api.testnet.shimmer.network')? // Insert your node URL here
+    .finish()
+    .await?;
+```
+
+#### Examples
+
+You can use the provided code [examples](sdk/examples) to get acquainted with the IOTA SDK. You can use the following command to run any example:
+
+```bash
+cargo run --example example_name --release
+```
+* Where `example_name` is the name from the [Cargo.toml](sdk/Cargo.toml) name from the examples section. For example:
+
+```bash
+cargo run --example node_api_core_get_info --release 
+```
+
+You can get a list of the available code examples with the following command:
+
+```bash
+cargo run --example
+```
+
+### API Reference
+
+#### Rust
+
+The IOTA SDK Rust API Reference is in the [crate documentation](https://docs.rs/iota-sdk/latest/iota_sdk/).
+
+## Contribute
+
+If you find any issues or have suggestions for improvements, please open an issue on the GitHub repository. You can also
+submit pull requests with bug fixes, new features, or documentation enhancements.
+
+Before contributing, please read and adhere to the [code of conduct](/.github/CODE_OF_CONDUCT.md).
+
+## License
+
+The IOTA SDK is open-source software licensed under Apache License 2.0. For more information, please read
+the [LICENSE](/LICENSE).
+
+## Acknowledgments
+
+The IOTA SDK project is built upon the contributions of many individuals and organizations. We would like to express our
+gratitude to the IOTA community and all the developers who have contributed to the project.
+
+For a complete list of contributors and acknowledgments, please refer to
+the [GitHub repository's contributor page](https://github.com/iotaledger/iota-sdk/graphs/contributors).
```

Comment:
This sections looks a bit useless to me, anyone can see this in GH anyways
Maybe we should have some ## Contributing section instead where we say that we're happy about issues/PRs?